src/containers/shared/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,81 @@
+import classnames from 'classnames'
+import { useCallback, useEffect, useRef, useState } from 'react'
+import ArrowIcon from '../../images/down_arrow.svg'
+import './dropdown.scss'
+
+export interface DropdownProps {
+  title: string | JSX.Element
+  children: any
+  className?: string
+}
+
+// TODO: Add useId after upgrading to react@18 to populate id on .dropdown-menu and aria-controlled by on .dropdown-toggle
```

Comment:
Might make sense to make an issue and link it in the comment (for react 18 upgrade)

Reply:
Created #668